src/token/erc1155/erc1155.cairo

```diff
@@ -65,33 +65,33 @@ mod ERC1155Component {
     #[derive(Drop, starknet::Event)]
     struct ApprovalForAll {
         #[key]
-        account: ContractAddress,
+        owner: ContractAddress,
         #[key]
         operator: ContractAddress,
         approved: bool
     }
 
     /// Emitted when the URI for token type `id` changes to `value`, if it is a non-programmatic URI.
     ///
-    /// If an `URI` event was emitted for `id`, the standard
-    /// https://eips.ethereum.org/EIPS/eip-1155#metadata-extensions[guarantees] that `value` will equal the value
+    /// If an `URI` event was emitted for `id`, the standard guarantees that `value` will equal the value
     /// returned by `IERC1155MetadataURI::uri`.
+    /// https://eips.ethereum.org/EIPS/eip-1155#metadata-extensions
     #[derive(Drop, starknet::Event)]
     struct URI {
+        value: ByteArray,
         #[key]
-        id: u256,
-        value: ByteArray
+        id: u256
     }
 
     mod Errors {
-        const INVALID_ACCOUNT: felt252 = 'ERC1155: invalid account';
         const INVALID_OPERATOR: felt252 = 'ERC1155: invalid operator';
```

Comment:
we're still not using this one aren't we?

Reply:
I hope is gone forever now :)

Reply:
🤞🤞